packages/ncids-js/src/components/nci-header/__tests__/mobile-menu/mobile-menu-adaptor-spanish.mock.ts

```diff
@@ -9,7 +9,6 @@ import {
 
 export class MockMobileMenuAdaptor implements MobileMenuAdaptor {
 	useUrlForNavigationId: boolean;
-	parentBackLabel = 'Espalda';
```

Comment:
LOL, how did I not see that it was literally back as in like a person's spine

Reply:
https://github.com/NCIOCPL/cgov-digital-platform/issues/3658